omero/developers/Python.rst

```diff
@@ -14,6 +16,19 @@ can be installed via ``pip``. We recommend you use a virtual environment:
     $ pip install omero-py==\ |version_py|
     Successfully installed Pillow-6.2.1 future-0.18.2 omero-py-5.6.1 zeroc-ice-3.6.5
 
+Install from Conda_:
+
+.. parsed-literal::
+
+    conda create -n myenv python=3.6
```

Comment:
You can simplify all this to one line
```suggestion
    conda create -n myenv -c ome python=3.6 omero-py
```

Reply:
You can optionally drop the `python=3.6` and you'll automatically get a version that's compatible with omero-py

Reply:
TIL :+1: